config/sentry.properties

```diff
@@ -0,0 +1,5 @@
+defaults.org=
```

Comment:
In CI we'll have to source the values from the environment.

https://docs.sentry.io/cli/configuration/#properties-files leaves me a bit unclear wether all of these can simply be set as env variables as well. Can you check that? In any case, it does not make sense to check-in this file.